synapse/cores/storage.py

```diff
@@ -87,6 +87,8 @@ def __init__(self,
         self.loadbus.on('core:save:del:rows:by:prop', self._loadDelRowsByProp)
         self.loadbus.on('core:save:set:rows:by:idprop', self._loadSetRowsByIdProp)
         self.loadbus.on('core:save:del:rows:by:idprop', self._loadDelRowsByIdProp)
+        self.loadbus.on('core:save:updateproperty', self._loadUpdateProperty)
+        self.loadbus.on('core:save:updatepropertyvalu', self._loadUpdatePropertyValu)
```

Comment:
small nit. these event names aren't consistent with the set/del versions.

Reply:
renamed